recipes/soqt/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.5)
+
+find_package(soqt CONFIG REQUIRED)
+
+if (SoQt_INCLUDE_DIRS)
+    MESSAGE(STATUS "SOQT_FOUND: " true)
+else()
+    MESSAGE(STATUS "SOQT_FOUND: " false)
+endif()
+
+MESSAGE(STATUS "SOQT_INCLUDE_DIR: " ${SoQt_INCLUDE_DIRS})
+MESSAGE(STATUS "SOQT_LIB_DIR: " ${SoQt_LIBRARY_DIRS})
+MESSAGE(STATUS "SOQT_VERSION: " ${SoQt_VERSION})
```

Comment:
I don't think that `cmake .` will fail if any of these values are not what is expected or if SoQt_INCLUDE_DIRS is empty.   It seems it will only fail if it `Findsoqt.cmake`  doesn't execute correctly